cmd/run.go

```diff
@@ -52,11 +57,36 @@ func newRunCMD(version string) *cobra.Command {
 				return err
 			}
 
-			client, err := resolver.PolicyReportClient()
+			discoveryClient, err := discovery.NewDiscoveryClientForConfig(k8sConfig)
 			if err != nil {
 				return err
 			}
 
+			var wgClient, orClient report.PolicyReportClient
+
+			_, err = discoveryClient.ServerResourcesForGroupVersion(orclient.OpenreportsReport.Group + "/" + orclient.OpenreportsReport.Version)
+			if err != nil {
+				logger.Error("openreports group not found in the cluster")
```

Comment:
is this really an error from a user perspective? Its somehow expected that only one api will be available